portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Oyin's Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+</head>
+
+<style>
+    body{
+        font: 22px 'Quicksand', sans-serif;
+        background-color: #a8d0e6;
+        }
+</style>
+
+
+<body>
+    <h1>"How Grace Hopper Impacted Me"</h1>
+    <h2>By: Oyin Sholeye</h2>
+    <h3>Posted on May 28, 2020</h3>
+    <img src="/images/IMG-3018.JPG" alt="GHC" style="width: 250px;height:300px" >
+    <p> Today, I would like to share how attending Grace Hopper was a very influencial experience for me.
+        Grace Hopper is a conference for women in computing.
+        One day, as I was sitting in my "Introduction to Computering Engineering" class trying to figure out 
+        an assignment, I was approached by one of the TA's. She told me that she noticed that I was in every 
+        computer science class and asked if I had considered attending the Grace Hopper Conference. She went on to tell
+        me that she got her internship offer to work at Facebook by going to the conference. I decided to apply 
+        for the GHC scholarship that my engineering school created. I didn't expect to be selected for the scholarship, but
+        I am so grateful that I was. MY school covered the hotel, flights, and ticket cost for me. I was super excited to 
+        have the chance to network with companies and meet new people.</p>
+
+        <p>Before arriving in Orlando for the conference, I expected to just go to the career fair and mind my 
+        business. I am so happy to say that I made so many new friends. </p>
+
+        <p>Day One of Grace Hopper was very overwhelming for me. I was in a space with 27,000 other girls. It also
+            gave me a sense of belonging because these were girls who shared the same passion for technolgy and coding as me.
+            I ended up meeting with a friend from high school while at the conference. It was really nice becuase we got to do a lot of catching
+            up.</p>
+        
+        <p> The main reasons why Grace Hopper was so influencial:</p>
+        <p> 1. I was able to interview with Google at the conference which led to me getting and accepting 
+            and internship offer for STEP.</p>
+        <p>2. I was able to build new friendships with some wonderful girls.</p>
+
+        <form>
+            <label for="comment"> Comments or Questions:</label><br>
+            <input type="text" id="comment" name="comment" >
+            <input type="submit" value="Post">
+        </form>
+</body>
+
+</html>
```

Comment:
File is missing a newline at the end

Reply:
Does that mean create a blank line on 54?

Reply:
Correct, https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline for more context